tests/schemas/api_v3/transactionsDryRun.schema.js

```diff
@@ -37,6 +38,7 @@ const event = {
 
 const eventSchemaWithSkipDecode = {
 	...event,
+	id: Joi.string().pattern(regex.HASH_SHA256).optional(),
```

Comment:
```suggestion
	id: Joi.string().pattern(regex.HASH_SHA256).required(),
```

Reply:
The `id` is not present when `skipDecode` is true. In our current implementation, we are assigning ids while decoding events.

Reply:
Then we'll need to verify the `formatEvent` implementation.

Reply:
Fixed implementation!